packages/bpk-docs/src/layouts/links.js

```diff
@@ -608,4 +608,15 @@ export default [
       },
     ],
   },
+  {
+    id: 'STYLE_GUIDE',
+    category: 'Style guide',
+    sort: true,
+    hero: {
+      url: `/${DesignTokensImage}`, // TODO
```

Comment:
It looks like you've got the image in this PR? (packages/bpk-docs/src/static/style_guide.jpg)

Reply:
That's different - the hero image and card images aren't exactly the same. E.g. we have `components.jpg` and `components_hero.jpg` too.